plugins/keepalive.koplugin/main.lua

```diff
@@ -0,0 +1,59 @@
+
+local ConfirmBox = require("ui/widget/confirmbox")
+local Device = require("device")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
+
+local KeepAlive = WidgetContainer:new{
+    name = "keepalive",
+}
+
+function KeepAlive:init()
+    if Device:isKobo() then
+        self.enable = function() UIManager:_stopAutoSuspend() end
```

Comment:
self.enable and self.disable can be initialized as a local function at the module level so you don't have to create two new anonymous functions every time `KeepAlive:init` is called. In fact, you can also initialize the menu callback based on the current device model only once.